sources/core/SaveableOperators.h

```diff
@@ -9,6 +9,28 @@
 namespace kv
 {
 
+template<long unsigned int N>
+inline unsigned int Hash(std::bitset<N>& value)
```

Comment:
Currently the build is failing because `SaveableOperators.h` doesn't contain `bitset` header.